app/templates/auth/create-user-error.html

```diff
@@ -0,0 +1,63 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Create contributor account – Digital Marketplace{% endblock %}
+
+{% block main_content %}
+
+{% if not token %}
+
+<header class="page-heading-smaller">
+  <h1>Expired link</h1>
+</header>
+<p>
+    The link you used to create an account may have expired.<br/>
+    Check you’ve entered the correct link or ask the person who invited you to send a new invitation.<br/>
+    If you still can’t create a contributor account, email <a href="mailto:enquiries@digitalmarketplace.service.gov.uk">enquiries@digitalmarketplace.service.gov.uk</a>.
+</p>
+
+{% elif not user.active %}
+
+<header class="page-heading-smaller">
+  <h1>Your account has been deactivated.</h1>
+</header>
+<p>
+    Email <a href="mailto:enquiries@digitalmarketplace.service.gov.uk">enquiries@digitalmarketplace.service.gov.uk</a> to reactivate your account.
+</p>
+
+{% elif user.locked %}
+
+<header class="page-heading-smaller">
+  <h1>Your account has been locked.</h1>
+</header>
+<p>
+    Email <a href="mailto:enquiries@digitalmarketplace.service.gov.uk">enquiries@digitalmarketplace.service.gov.uk</a> to unlock your account.
+</p>
+
+{% elif user %}
+
+  {% if user.role == 'supplier' and token.get("supplier_id") != user.supplier_id %}
+
+  <header class="page-heading-smaller">
+    <h1>The details you provided are registered with another supplier.</h1>
+  </header>
+  <p>You were invited by ‘{{ token.get("supplier_name") }}’</p>
+  <p>Your account is registered with ‘{{ user.supplier_name }}’</p>
+  <br>
+  <p>
+    <a href="{{ url_for('.render_login') }}">Log in</a> to your {{ user.supplier_name }} account or email
+    <a href="mailto:enquiries@digitalmarketplace.service.gov.uk">enquiries@digitalmarketplace.service.gov.uk</a> to register your account with a different organisation.
+  </p>
+
+  {% else %}
+
+  <header class="page-heading-smaller">
+    <h1>Account already exists</h1>
+  </header>
+
+  <p>
+    <a href="{{ url_for('.render_login') }}">Log in</a> to your account.
```

Comment:
An odd thing here is if you click this link while logged in with a different account from the one you're trying to create then you get taken straight to the supplier dashboard as the logged-in user, with no chance to swap to the new user account.

But this is an edge case only likely to be encountered by one of us testing login stuff, so fine I think.